TestFramework/ExtensionMethods.cs

```diff
@@ -335,5 +340,55 @@ public static void SendKeys(IWebDriver driver, string keys)
             action.SendKeys(keys);
             action.Perform();
         }
+
+        public static void OpenNewPage(IWebDriver webdriver, String url)
+        {
+           
+            try
+            {
+                ((IJavaScriptExecutor)webdriver).ExecuteScript("window.open();");
+                webdriver = webdriver.SwitchTo().Window(webdriver.WindowHandles.Last());
+                webdriver.Navigate().GoToUrl(url); 
+            }
+            catch (Exception ex)
+            {
+                Logger.Error(ex, $"Cannot Move to element By locator:");
+               
+            }
+
+        }
+
+        public static void NavigateUrl(IWebDriver webdriver, String url)
```

Comment:
Do we need a new method for this one line statement?

Reply:
Removed it.